src/sass/app_2/_common/components/calendar.scss

```diff
@@ -115,7 +115,7 @@
         }
     }
     &__text {
-        @include typeface(--paragraph-left-normal-black);
+        @include typeface(--small-center-normal-black);
```

Comment:
could you add `unset` as the 2nd argument? `typeface(--small-center-normal-black, unset);`